docs/apis/streaming/connectors/kinesis.md

```diff
@@ -299,3 +299,43 @@ Otherwise, the returned stream name is used.
 Other optional configuration keys can be found in `KinesisConfigConstants`.
 		
 		
+### Using Non-AWS Kinesis Endpoints for Testing
+
+It is sometimes desirable to have Flink operate as a consumer or producer against a non-AWS Kinesis endpoint such as kinesalite; this is especially useful when performing functional testing of a Flink application. The AWS endpoint that would normally be inferred by the AWS region set in the Flink configuration must be overriden via a configuration property.
```

Comment:
(original inline comment was marked outdated. Copying the comment again, just in case we miss it ;) )

We should be linking "kinesalite" to the project. Should we also capitalize "K"?

However, I'm not sure if it's a good idea to refer to another public Github project in the documentation, even though Kinesalite is a widely adopted solution for local Kinesis testing. We'll need to have @rmetzger 's comment on this.